### PR TITLE
[16.0][ADD] stock_release_channel_show_volume/weight

### PR DESCRIPTION
**Pull request by jbaudoux**
* stock_release_channel: Dashboard beautification + allow to show/hide last done delivery
  * Clean display using Flex box
  * Transfers button moved on progress bar
  * Use kanban progress instead of form progressbar widget
  * Plan shipment button moved to 3-dots actions
  * Added margin on action labels
  * Display of "Last picking done transfer" is now configurable
* stock_release_channel_show_volume: Show volumes on release channels
* stock_release_channel_show_weight: Show weight on release channels

### Before
image

### After
image image

cc @mt-software-de @sebalix

### CODE REVIEW

**Comment by jbaudoux** [comment]
The counters in the below section should be adapted (not part of this PR):
* Waiting: Count only first picking of the chain in waiting availability, so what is waiting a replenishment.
* Late: Count only first picking (so take ready or waiting), so either pick is late, or pack is late if pick is done. In a pick/pack/ship, we don't want to count 3 times late for a single SO.
* Priority: Count only first  (so take ready or waiting) picking, so either pick is late, or pack is late if pick is done. In a pick/pack/ship, we don't want to count 3 times late for a single SO.

**Comment by sebalix** [comment]
Build fixed here: https://github.com/OCA/wms/pull/849

**Comment by jbaudoux** [comment]
Other build fix here: https://github.com/OCA/wms/pull/850

**Comment by jbaudoux** [comment]
Other build fix here https://github.com/OCA/wms/pull/851

**Comment by jbaudoux** [comment]
rebased

File: stock_release_channel_show_volume/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+{
+    "name": "Stock Release Channels show Volume",
+    "summary": "Display volumes of stock release channels",
+    "version": "16.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "MT-Software, BCIM, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/wms",
+    "depends": [
+        "stock_picking_volume",
```
Comment (sebalix) [inline stock_release_channel_show_volume/__manifest__.py:12]: About `stock_picking_volume`, I forward-ported the work done by @mt-software-de on 14.0 here:
- https://github.com/OCA/stock-logistics-warehouse/pull/1954

File: stock_release_channel_show_weight/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Show the volume on the release channels
```
Comment (sebalix) [inline stock_release_channel_show_weight/readme/DESCRIPTION.rst]:
```suggestion
Show the weight on the release channels
```

**Review by sebalix — APPROVED** [review]
Commit 47e8646d2a3631d58d605a295f83729f211395be can be squashed in `[ADD] stock_release_channel_show_volume`

File: stock_release_channel/models/stock_release_channel.py [hunk 3]
```diff
@@ -1,7 +1,10 @@
 # Copyright 2020 Camptocamp
+# Copyright 2024 Jacques-Etienne Baudoux (BCIM) <je@bcim.be>
```
Comment (mt-software-de) [inline stock_release_channel/models/stock_release_channel.py]: Could you please add my copyright, i forgot to add it. in https://github.com/OCA/wms/pull/699
```suggestion
# Copyright 2023 Michael Tietz (MT Software) <mtietz@mt-software.de>
# Copyright 2024 Jacques-Etienne Baudoux (BCIM) <je@bcim.be>
```

**Review by mt-software-de — APPROVED** [review]
LG, could you please add my Copyright

File: stock_release_channel/models/stock_release_channel.py (hunk 3 above)
Comment (jbaudoux) [reply stock_release_channel/models/stock_release_channel.py]: done

**Pull request merged**